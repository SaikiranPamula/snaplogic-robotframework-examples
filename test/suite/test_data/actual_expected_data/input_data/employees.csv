name,role,salary
Swapna,Engineer,90000
Bob,Analyst,75000