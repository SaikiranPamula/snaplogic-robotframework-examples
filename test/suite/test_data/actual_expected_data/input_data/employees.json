[
  {
    "name": "Swapna",
    "role": "Engineer",
    "salary": 90000
  },
  {
    "name": "Bob",
    "role": "Analyst",
    "salary": 75000
  }
]