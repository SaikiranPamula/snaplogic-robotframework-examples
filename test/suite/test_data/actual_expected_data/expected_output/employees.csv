"id","name","role","salary"
"1","Swapna","Engineer","90000"
"2","Bob","Analyst","75000"